tamr_toolbox/dataset/manage.py

```diff
@@ -0,0 +1,231 @@
+from typing import List, Optional, Union
+
+from tamr_unify_client import Client
+from tamr_unify_client.dataset.resource import Dataset
+from tamr_unify_client.attribute.resource import AttributeSpec
+from tamr_unify_client.attribute.type import AttributeType
+
+from tamr_toolbox.models.data_type import JsonDict
+
+
+def create(
+    *,
+    client: Client,
+    dataset_name: str,
+    primary_keys: Optional[List[str]] = None,
```

Comment:
Since Primary keys are not optional this could be: 
```suggestion
    primary_keys: [List[str]] = None,
```